<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load</string>
  </property>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>881</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Parameters</string>
    </attribute>
    <widget class="QLabel" name="vn_kv_label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>14</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Active Power</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="p_kw">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Reactive Power</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="q_kvar">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="vn_kv_label_2">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>15</y>
       <width>121</width>
       <height>10</height>
      </rect>
     </property>
     <property name="text">
      <string>kW</string>
     </property>
    </widget>
    <widget class="QLabel" name="vn_kv_label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>44</y>
       <width>121</width>
       <height>10</height>
      </rect>
     </property>
     <property name="text">
      <string>kVar</string>
     </property>
    </widget>
    <widget class="QComboBox" name="bus">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="name_label_2">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bus</string>
     </property>
    </widget>
    <widget class="QLabel" name="vn_kv_label_5">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>114</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Documentation</string>
    </attribute>
    <widget class="QWebView" name="webView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>651</width>
       <height>391</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>http://pandapower.readthedocs.io/en/v1.3.0/elements/load.html#load</string>
      </url>
     </property>
    </widget>
    <widget class="QWebView" name="webView_2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>201</width>
       <height>391</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>http://pandapower.readthedocs.io/en/v1.3.0/_images/load.png</string>
      </url>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
